src/bnb_trading/ichimoku_module.py

```
@@ -220,143 +220,142 @@ class IchimokuAnalyzer:
     def fetch_ichimoku_data(self, interval: str = "1d", limit: int = 100):
         """Fetch data for Ichimoku analysis"""
         try:
-            params = {
-                "symbol": "BNBUSDT",
-                "interval": interval,
-                "limit": min(limit, 1000)
-            }
-            
+            params = {"symbol": "BNBUSDT", "interval": interval, "limit": min(limit, 1000)}
+
             response = requests.get(f"{self.base_url}/klines", params=params)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)